### PR TITLE
Add Support for Java 8u171/172

### PR DESCRIPTION
**Pull request by mtron**

### CODE REVIEW

**Comment by dalesit** [comment]
Ah great, already been done. It's that time of year again... merge ASAP please, and update release on the forge...

**Comment by dalesit** [comment]
"This JRE (version 8u172) will expire with the release of the next critical patch update scheduled for July 17, 2018" so make a note in your diaries... would be nice if we could get the next version released to the forge in advance of the expiry date...

File: manifests/init.pp [hunk 1]
```diff
@@ -197,7 +197,7 @@
   }
 
   # define package name
-  if versioncmp($version_final, '8u151') >= 0 and $maj_version != '9' {
+  if versioncmp($version_final, '8u172') >= 0 and $maj_version != '9' {
```
Comment (antoineco) [inline manifests/init.pp]: Will break all versions below 8u172, please revert this particular change.

**Review by antoineco — CHANGES_REQUESTED** [review]

**Comment by mtron** [comment]
> Will break all versions below 8u172, please revert this particular change.

ups. sorry. Reverted

File: manifests/init.pp (hunk 1 above)
Comment (mtron) [reply manifests/init.pp]: ups. sorry. Reverted

**Review by antoineco — APPROVED** [review]

**Comment by antoineco** [comment]
Thanks @mtron! 🙌

**Comment by antoineco** [comment]
Released in [v2.9.4](https://forge.puppet.com/aco/oracle_java/2.9.4/changelog)

**Comment by dalesit** [comment]
Many thanks to you both👏

**Pull request merged**